### ISSUE TITLE
Possible to Use This Receiver Instead?

### ISSUE THREAD

**Issue by AlexLandherr** [mitxela/PrecisionClock-Hardware#2]
Would [this](https://gnss.store/lea-m8f-timing-gnss-modules/43-elt0030.html) GNSS receiver work with your clock kit?

I know it's a very fancy module but I'm curious whether it would work or not.

On the linked to website there's also [several receivers](https://gnss.store/43-neo-m8t-timing-gnss-modules) based on the NEO-M8T; would these work too?

If my questions seem like they have obvious answers it's probably because my skills are in programming on regular computers (PCs and servers); not electrical engineering.

**Comment by mitxela** [comment]
Yes, pretty much any newer generation of ublox module will work. Most of them are still configured for the correct NMEA baud out of the box, but you can customise their output if not.

I've used the 8th gen ublox modules with the clock, they work very well. One thing to mention is that although they support broadband antennas that can pick up GLONASS, Baidu, etc simultaneously, this is used to assist navigation but not to generate the time pulse, that's still purely from GPS as I understand it.

Also, the clock display has a jitter of a few hundred microseconds, so the difference in performance of these timepulse modules is kind of irrelevant. The only reason to use one would be if you need it for something else, and want the clock to run from the same thing, which is quite easy to do. Just feed it the PPS and the NMEA data.
